Source/Core/Common/Common.h

```diff
@@ -39,9 +39,13 @@ extern const std::string scm_distributor_str;
 
 // Since they are always around on Windows
 #define HAVE_WX 1
+#ifndef HAVE_OPENAL
```

Comment:
I believe the less logic or defines in the build system, the better. Also, it's quite annoying to edit it 😢